src/DoctrineORMModule/Service/MigrationsConfigurationFactory.php

```diff
@@ -48,6 +48,10 @@ public function __invoke(ContainerInterface $container, $requestedName, array $o
             $configuration->setMigrationsColumnName($migrationsConfig['column']);
         }
 
+        if (method_exists($configuration, 'setCustomTemplate')) {
+            $configuration->setCustomTemplate($migrationsConfig['custom_template']);
```

Comment:
I think in order to cover all the bases you need to verify `custom_template` exists on the `$migrationsConfig` before setting it.

Reply:
Your'e right! i added a check to ensure `custom_template` exists of the `$migrationsConfig` before accessing it.